widget/embedded/src/pages/WalletsPage.tsx

```diff
@@ -11,11 +11,10 @@ import { useAppStore } from '../store/AppStore';
 import { getContainer } from '../utils/common';
 
 const ListContainer = styled('div', {
-  display: 'flex',
-  justifyContent: 'space-evenly',
-  alignItems: 'center',
+  display: 'grid',
   gap: '$10',
-  flexWrap: 'wrap',
+  gridTemplateColumns: ' repeat(auto-fill, minmax(100px, 1fr))',
+  alignContent: 'baseline',
   paddingTop: '$5',
   height: '100%',
 });
```

Comment:
To resolve the issue and maintain consistent sizes for the wallet button and its skeleton, you can retain the Flexbox styles and simply reduce the width of the wallet loading button within the UI package.